vitumob/controllers/rates.py

```diff
@@ -2,47 +2,51 @@
 
 import os
 import math
-from datetime import datetime
-
-from flask import Blueprint, Response
-from google.appengine.ext import ndb
-
+import bson
```

Comment:
This should in it's own PR. Since we are changing 1 file/controller and its models at a time.

Reply:
okay will create its own PR